lib/cadet_web/router.ex

```diff
@@ -28,6 +28,21 @@ defmodule CadetWeb.Router do
   # Authenticated Pages
   scope "/", CadetWeb do
     pipe_through([:api, :auth, :ensure_auth])
+
+    scope "/answer", Answer do
+      get("/programming/:id", ProgrammingAnswerController, :show)
+      post("/programming", ProgrammingAnswerController, :submit)
+      get("/mcq/:id", MCQAnswerController, :show)
+      post("/mcq", MCQAnswerController, :submit)
+    end
```

Comment:
As I understand it, question IDs are only unique within a mission, so this needs a mission ID somewhere.

Also, can we have one path for all questions, no matter the type (programming or mcq)? In that case, you could nest this under the current `/missions` path as the POST and GET at `/missions/{missionId}/{questionId}`.

Reply:
question IDs should be unique across all but anyway this AnswerController needs a rework to unify the two kinds of questions anyway.